### PR TITLE
Describe how to access to a title extension from a template

### PR DESCRIPTION
**Pull request by romolox**
I wrote how to acces to a title extension within a template since I figure it out from the code and I tought it should be better to have it described in the documentation. My english is really bad so I don't know if  what I wrote on the doc is understandable! :)

### CODE REVIEW

**Comment by yakky** [comment]
LGTM
@evildmp ?

File: docs/how_to/extending_page_title.rst [hunk 1]
```diff
@@ -200,6 +200,10 @@ page extension to every page, a page may not have the ``iconextension``
 relationship available, hence the use of the ``{% if ... %}...{% endif %}``
 above.
 
+In order to access to a title extension in page templates you will need to access the title object with the
+get_title_obj function:
```
Comment (evildmp) [inline docs/how_to/extending_page_title.rst]: I would express this like this:

```
In order to access a title extension within a template, get the ``Title`` object using
``request.current_page.get_title_obj``, for example::

    {{ request.current_page.get_title_obj.your_title_extension }}
```

I would also like to see sub-headings within this section describing how to use extensions in templates:

    Page extensions
    ---------------

and 

    Title extensions
    ----------------

**Comment by romolox** [comment]
@evildmp @yakky Should be ok now

**Comment by yakky** [comment]
LGTM waiting for test to finish before merging

**Pull request merged**